featuretools/primitives/standard/transform_primitive.py

```diff
@@ -186,7 +186,7 @@ class Minute(TransformPrimitive):
     """
     name = "minute"
     input_types = [ColumnSchema(logical_type=Datetime)]
-    return_type = ColumnSchema(logical_type=Ordinal(range(59)), semantic_tags={'category'})
+    return_type = ColumnSchema(logical_type=Ordinal(list(range(59))), semantic_tags={'category'})
```

Comment:
for consistency's sake, this and line 165 (the Second primitive) should probably specify the `order=` for the parameter passed in